doc/start/documenting-ceph.rst

```diff
@@ -105,37 +106,36 @@ approach. You must:
 
 #. Fork the Ceph project. See https://github.com/ceph/ceph.
 
-#. Clone your fork of the Ceph project to your local host.
+#. Clone your fork of the Ceph project to your local host. This creates what is
+   known as a "local working copy".
 
+The Ceph documentation is organized by its components:
```

Comment:
s/its//

Reply:
I think we can go one better here and just say "The Ceph documentation is organized by component:".

So that's what I'm going to do.